src/app.js

```diff
@@ -22,4 +22,8 @@ window.addEventListener('message', ({ data: req }) => {
     app.$set({ enabled: true });
     searchValue.set(req.data);
   }
-});
\ No newline at end of file
+});
+
+window.addEventListener('click', (e) => {
+  if(e.target.className.includes("search-wrapper")) close()
```

Comment:
@asharonbaltazar Can we change this to `e.target.classList.contains("search-wrapper")`?  
It seems more appropriate here.

Reply:
For the sake of learning, can I ask why?

Reply:
Sure!
- `className` is a string. And  `includes` is used to check whether substring is present in the string. If let's say some element had a class `search-wrapper-abc`, then the `includes` check would return true as it would match `search-wrapper` as substring.
- `classList` is similar to an array, and has a `contains` method that would check if the *exact* class name string is present on the element instead of checking for substring.

It might not give different results in our use case, but if later on there was some element with class having `search-wrapper` in some part of the name, it would cause issues.

Hope this helps :)

Reply:
It does! Updated the condition.